tools/merger.py

```diff
@@ -5,32 +5,35 @@
 
 # todo: can speed up by using info about how many halos instead concatenating
 
-def extract_superslab(fns):
-    z_in = extract_redshift(fns)
-    return np.array([np.int(fn.split('z%.3f.'%z_in)[1].split('.asdf')[0]) for fn in fns])
-    
-def extract_redshift(fns):
-    redshift = float(fns[0].split('z')[-1][:5])
+
+def extract_superslab(fn):
+    # looks like "associations_z0.100.0.asdf"
+    return int(fn.split('.')[-2])
```

Comment:
I think this can be simplified to what I've written?  When writing these string parsers, I find it very helpful to leave a command showing the format of the string that you have in mind while writing the parser.  Did I get it right, here?

Reply:
hahaha, yes, thank you!